guide/guide-qip.rst

```diff
@@ -0,0 +1,264 @@
+.. QuTiP 
+   Copyright (C) 2011-2012, Paul D. Nation & Robert J. Johansson
+
+.. _qip:
+
+*********************************************
+Quantum Information Processing
+*********************************************
+
+Introduction
+============
+
+The Quantum Information Processing (QIP) module aims at providing basic tools for quantum computing simulation both for simple quantum algorithm design and for experimental realization. It offers two different approaches, one with :class:`qutip.qip.QubitCircuit` calculating unitary evolution under quantum gates by matrix production, another called :class:`qutip.qip.device.Processor` using open system solver in QuTiP to simulate NISQ device.
```

Comment:
Great intro. Please add the explicit form of the NISQ acronym when first citing it.